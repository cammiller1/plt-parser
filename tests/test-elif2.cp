int x;
x = 2;
if (x > 10) {
	prints("went to if branch");
}
else if (x == 1) {
	prints("went to elif branch");
}
else if (x == 2) {
	prints("went to second elif branch");
} else {
	prints("went to else branch");
}