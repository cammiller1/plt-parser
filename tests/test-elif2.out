went to second elif branch